build-support/bin/rust/calculate_engine_hash.sh

```diff
@@ -33,15 +32,13 @@ function calculate_current_hash() {
     cd "${REPO_ROOT}" || exit 1
     (
       echo "${MODE_FLAG}"
-      echo "${RUST_TOOLCHAIN_CONTENTS}"
       uname -mps
       # the engine only depends on the implementation and major.minor version, not the patch
       "${PY}" -c 'import sys; print(sys.implementation.name, sys.version_info.major, sys.version_info.minor)'
       git ls-files --cached --others --exclude-standard \
         "${NATIVE_ROOT}" \
-        "${REPO_ROOT}/rust-toolchain" \
         "${REPO_ROOT}/build-support/bin/rust" |
-        grep -v -E -e "/BUILD$" -e "/[^/]*\.md$" |
+        grep -v -E -e "/BUILD$" -e "/[^/]*\.md$" -e "/[^/]*\.sh$" |
```

Comment:
Hm... why would we exclude shell scripts here? In particular, that would cause this script to fail to detect changes to itself.

Reply:
I was trying to exclude the `_shim.sh` script because I kept rebuiding the engine while mucking with it